Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionMenuItemCocoa.mm

```diff
@@ -298,8 +298,8 @@ + (BOOL)usesUserKeyEquivalents
         result.keyEquivalentModifierMask = command->modifierFlags().toRaw();
     }
 
-    auto idealSize = CGSizeMake(16, 16);
-    auto *image = icon(idealSize);
+    auto idealSize = WebCore::FloatSize(16, 16);
+    auto *image = icon(idealSize) ? icon(idealSize)->image().get() : nil;
```

Comment:
This should save the result of `icon(idealSize)` and use it instead of calling it again (it can be expensive).